my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
+import os
+from astropy.io import fits
+import glob
+data = fits.getdata('/Users/bostroem/Desktop/images/hubble_img.fits')
+col1 = data[20, :] 
+col2 = data[200, :]
+col3 = data[800, :]
+pyplot.figure()
+pyplot.plot(np.arange(1024), col1)
```

Comment:
I prefer including the name for the data series here like:

`pyplot.plot(np.arange(1024), col1, label='Column 20')`

`Column 20` is not a very descriptive name.